packages/csv-parser-price/src/main.js

```diff
@@ -183,9 +183,11 @@ export default class CsvParserPrice {
 // Easiest way to wrap the getCustomTypeDefinition in the memoize method
 CsvParserPrice.prototype.getCustomTypeDefinition = memoize(
   function _getCustomTypeDefinition (customTypeKey) {
-    const getTypeByKeyUri = createRequestBuilder().types
+    const getTypeByKeyUri = createRequestBuilder({
```

Comment:
A more appropriate reason can be found [here](https://github.com/lerna/lerna#how-bootstrap-works)

Lerna works by symlinking internal modules when you do 'npm install', so when you make a change to a module, all other modules get's the new updated change, and this is the desired behaviour